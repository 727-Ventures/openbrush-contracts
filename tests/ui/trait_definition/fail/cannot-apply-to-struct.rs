#[openbrush::trait_definition]
pub struct NotTrait;

fn main() {}