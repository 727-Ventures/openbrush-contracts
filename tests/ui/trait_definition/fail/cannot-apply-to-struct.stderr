error: custom attribute panicked
 --> tests/ui/trait_definition/fail/cannot-apply-to-struct.rs:1:1
  |
1 | #[openbrush::trait_definition]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: called `Result::unwrap()` on an `Err` value: Error("expected `trait`")
